src/common/urlUtils.ts

```diff
@@ -9,6 +9,20 @@ import Cdp from '../cdp/api';
 import { escapeRegexSpecialChars } from './sourceUtils';
 import { AnyChromeConfiguration } from '../configuration';
 
+let isCaseSensitive = process.platform !== 'win32';
```

Comment:
This again 😁 @digeff

Long history, we need to be careful to be case insensitive, but case "preserving", meaning we send back the correct case to vscode and set breakpoints with the correct case. And in v1 we set the bps with a case-insensitive regex.

Reply:
Duly noted! For now we just use this for the micromatch test, but it's good to know that we'll need to be careful around this going forward.